src/main/java/com/box/sdk/BoxStoragePolicy.java

```diff
@@ -0,0 +1,162 @@
+package com.box.sdk;
+
+import java.net.URL;
+
+import com.eclipsesource.json.JsonObject;
+import com.eclipsesource.json.JsonValue;
+import com.eclipsesource.json.ParseException;
+
+/**
+ *  Represents a BoxStoragePolicy.
+ */
+@BoxResourceType("BoxStoragePolicy")
+public class BoxStoragePolicy extends BoxResource {
+
+    /**
+     * Storage Policies URL Template.
+     */
+    public static final URLTemplate STORAGE_POLICY_URL_TEMPLATE = new URLTemplate("storage_policies");
+
+    /**
+     * Storage Policies URL Template.
+     */
+    public static final URLTemplate STORAGE_POLICY_WITH_ID_URL_TEMPLATE = new URLTemplate("storage_policies/%s");
+
+    /**
+     * The default limit of entries per response.
+     */
+    private static final int DEFAULT_LIMIT = 100;
+
+    /**
+     * Constructs a BoxStoragePolicy with a given ID.
+     *
+     * @param api the API connection to be used by the BoxStoragePolicy.
+     * @param id  the ID of the BoxStoragePolicy.
+     */
+    public BoxStoragePolicy(BoxAPIConnection api, String id) {
+        super(api, id);
+    }
+
+    /**
+     * Gets information for a Box Storage Policy with optional fields.
+     *
+     * @param fields the fields to retrieve.
+     * @return info about this item containing only the specified fields, including storage policy.
+     */
+    public BoxStoragePolicy.Info getInfo(String... fields) {
+        QueryStringBuilder builder = new QueryStringBuilder();
+        if (fields.length > 0) {
+            builder.appendParam("fields", fields);
+        }
+        URL url = STORAGE_POLICY_WITH_ID_URL_TEMPLATE.buildWithQuery(this.getAPI().getBaseURL(), builder.toString(),
+                this.getID());
+
+        BoxAPIRequest request = new BoxAPIRequest(this.getAPI(), url, "GET");
+        BoxJSONResponse response = (BoxJSONResponse) request.send();
+        return new Info(response.getJSON());
+    }
+
```

Comment:
We should probably also add an `assign()` method to this class, otherwise it's not very useful to have instances